# Directing resources to address COVID-19 health inequities

BA775 B04 
Fanfei Zhao, Sixuan Wang, Shu Wang, Yuesen Zhang, Jiadai Yu

Our goal is to establish a strategy for equal distribution of health care resources for the treatment of COVID-19. We will analyze the impact of inequities on community health status, find the hardest hit areas, and map out how to allocate medical resources (vaccines, tests, treatments, etc.) equally and to efficiently protect the safety of all communities.

Source: California Health and Human Services (https://data.chhs.ca.gov/dataset/covid-19-equity-metrics)

## 1. Data Overview and Data Dictionary

### 1.1 covid-19-case-rate-by-social-det 

In [6]:
%%bigquery
SELECT *
FROM `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-case-rate-by-social-det`

Downloading: 100%|██████████| 7908/7908 [00:01<00:00, 6201.63rows/s]


date         social_det    social_tier  sort  \
0     2022-08-08             income    $60k - $80k   3.0   
1     2022-08-08             income   $80k - $100k   2.0   
2     2022-08-01             income     below $40K   5.0   
3     2022-07-18             income    above $120K   0.0   
4     2022-07-05             income  $100k - $120k   1.0   
...          ...                ...            ...   ...   
7903  2022-08-22  income_cumulative    $60k - $80k   NaN   
7904  2022-08-29  income_cumulative    $60k - $80k   NaN   
7905  2022-09-06  income_cumulative    $60k - $80k   NaN   
7906  2022-09-13  income_cumulative    $60k - $80k   NaN   
7907  2022-09-21  income_cumulative    $60k - $80k   NaN   

      cases_7dayavg_7daysago  population  case_rate_per_100k  \
0               3.060286e+03     8648044           35.387028   
1               1.820143e+03     5576687           32.638426   
2               3.060000e+03     7298172           41.928308   
3               9.868571e+02     2748206           35.909140   
4               1.300714e+03     3270993           39.765120   
...                      ...         ...                 ...   
7903            2.088642e+06     8630488        24200.740445   
7904            2.100823e+06     8630488        24341.879625   
7905            2.110763e+06     8630488        24457.052718   
7906            2.119410e+06     8630488        24557.244040   
7907            2.126021e+06     8630488        24633.844575   

      state_case_rate_per_100k  case_rate_per_100k_30_days_ago  \
0                    35.120880                       44.130515   
1                    35.120880                       43.969226   
2                    41.334032                       31.796647   
3                    43.132867                       40.319590   
4                    38.335856                       42.157264   
...                        ...                             ...   
7903              25214.856229                    23434.862548   
7904              25353.501045                    23692.947606   
7905              25467.525361                    23931.346640   
7906              25566.049725                    24110.270474   
7907              25639.260827                    24293.249698   

      rate_diff_30_days  
0             -8.743488  
1            -11.330800  
2             10.131660  
3             -4.410450  
4             -2.392145  
...                 ...  
7903         765.877897  
7904         648.932019  
7905         525.706078  
7906         446.973566  
7907         340.594877  

[7908 rows x 10 columns]

In [9]:
%%bigquery
SELECT DISTINCT social_det, social_tier, sort
FROM `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-case-rate-by-social-det`
ORDER BY social_det, sort;

Downloading: 100%|██████████| 24/24 [00:01<00:00, 18.91rows/s]


social_det    social_tier  sort
0            crowding   less than 2%   0.0
1            crowding        2% - 5%   1.0
2            crowding       5% - 10%   2.0
3            crowding      10% - 15%   3.0
4            crowding      15% - 20%   4.0
5            crowding       over 20%   5.0
6              income    above $120K   0.0
7              income  $100k - $120k   1.0
8              income   $80k - $100k   2.0
9              income    $60k - $80k   3.0
10             income    $40k - $60k   4.0
11             income     below $40K   5.0
12  income_cumulative    above $120K   NaN
13  income_cumulative    $40k - $60k   NaN
14  income_cumulative   $80k - $100k   NaN
15  income_cumulative     below $40K   NaN
16  income_cumulative  $100k - $120k   NaN
17  income_cumulative    $60k - $80k   NaN
18          insurance   less than 5%   0.0
19          insurance       5% - 10%   1.0
20          insurance      10% - 15%   2.0
21          insurance      15% - 25%   3.0
22          insurance      25% - 35%   4.0
23          insurance       over 35%   5.0

The covid-19-case-rate-by-social-det table contains 10 fields and 7908 records, that contains information of covid-19 case rate and social determinants of health.

### 1.2 covid-19-demographic-data-completeness

In [10]:
%%bigquery
SELECT *
FROM `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-demographic-data-completeness`;

Downloading: 100%|██████████| 144348/144348 [00:01<00:00, 124509.50rows/s]


county       sogi_category  metric  missing  not_missing    total  \
0           Orange     gender_identity   cases    110.0      16810.0  16920.0   
1           Orange  sexual_orientation   cases  16169.0        751.0  16920.0   
2           Orange  sexual_orientation  deaths     85.0          1.0     86.0   
3           Placer  sexual_orientation   cases   1257.0        250.0   1507.0   
4           Plumas  sexual_orientation   cases     51.0         16.0     67.0   
...            ...                 ...     ...      ...          ...      ...   
144343        Mono  sexual_orientation  deaths      1.0          0.0      1.0   
144344      Nevada  sexual_orientation  deaths      1.0          0.0      1.0   
144345  San Benito  sexual_orientation  deaths      1.0          0.0      1.0   
144346      Solano  sexual_orientation  deaths      1.0          0.0      1.0   
144347      Tehama  sexual_orientation  deaths      1.0          0.0      1.0   

        percent_complete  percent_complete_30_days_prior  \
0               0.993499                        0.995326   
1               0.044385                        0.046026   
2               0.011628                        0.000000   
3               0.165893                        0.187569   
4               0.238806                        0.248649   
...                  ...                             ...   
144343          0.000000                             NaN   
144344          0.000000                        0.000000   
144345          0.000000                        0.500000   
144346          0.000000                        0.000000   
144347          0.000000                        0.000000   

        percent_complete_30_days_diff report_date  
0                           -0.001827  2022-08-30  
1                           -0.001640  2022-08-30  
2                            0.011628  2022-08-30  
3                           -0.021677  2022-08-30  
4                           -0.009843  2022-08-30  
...                               ...         ...  
144343                            NaN  2020-12-07  
144344                       0.000000  2020-12-07  
144345                      -0.500000  2020-12-07  
144346                       0.000000  2020-12-07  
144347                       0.000000  2020-12-07  

[144348 rows x 10 columns]

The covid-19-demographic-data-completeness is a supporting table that provides information about the completeness of the data in other tables, and can be used to check the validity of the analysis results.

### 1.3 covid-19-demographic-rate-cumulative 

In [11]:
%%bigquery
SELECT * 
FROM `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-demographic-rate-cumulative`;

Downloading: 100%|██████████| 2986/2986 [00:01<00:00, 2388.11rows/s]


date      county demographic_set  \
0     2022-09-22  California        Combined   
1     2022-09-22  California        Combined   
2     2022-09-22  California        Combined   
3     2022-09-22     Alameda         age_gp4   
4     2022-09-22   Calaveras         age_gp4   
...          ...         ...             ...   
2981  2022-09-22    Tuolumne  race_ethnicity   
2982  2022-09-22    Tuolumne  race_ethnicity   
2983  2022-09-22     Ventura  race_ethnicity   
2984  2022-09-22        Yolo  race_ethnicity   
2985  2022-09-22        Yuba  race_ethnicity   

                        demographic_set_category  metric  metric_value  \
0                                      Statewide   cases       98880.0   
1                                      Statewide  deaths         419.0   
2                                      Statewide   tests     1414806.0   
3                                           None   tests          51.0   
4                                           None   tests          14.0   
...                                          ...     ...           ...   
2981  Native Hawaiian and other Pacific Islander   cases           0.0   
2982  Native Hawaiian and other Pacific Islander  deaths           0.0   
2983  Native Hawaiian and other Pacific Islander  deaths           0.0   
2984  Native Hawaiian and other Pacific Islander  deaths           0.0   
2985  Native Hawaiian and other Pacific Islander  deaths           0.0   

      metric_value_per_100k applied_suppression  population_percentage  \
0                246.404360                None             100.000000   
1                  1.044129                None             100.000000   
2               3525.630738                None             100.000000   
3                       NaN                None                    NaN   
4                       NaN                None                    NaN   
...                     ...                 ...                    ...   
2981               0.000000                None               0.236863   
2982               0.000000                None               0.236863   
2983               0.000000                None               0.170860   
2984               0.000000                None               0.459278   
2985               0.000000                None               0.392231   

      metric_total_percentage  metric_value_30_days_ago  \
0                  100.000000                  225409.0   
1                  100.000000                    1176.0   
2                  100.000000                 1793924.0   
3                    0.041642                      58.0   
4                    0.672430                      22.0   
...                       ...                       ...   
2981                 0.000000                       NaN   
2982                      NaN                       0.0   
2983                 0.000000                       0.0   
2984                      NaN                       0.0   
2985                      NaN                       0.0   

      metric_value_per_100k_30_days_ago  \
0                            561.708742   
1                              2.930537   
2                           4470.375159   
3                                   NaN   
4                                   NaN   
...                                 ...   
2981                                NaN   
2982                           0.000000   
2983                           0.000000   
2984                           0.000000   
2985                           0.000000   

      metric_value_per_100k_delta_from_30_days_ago  \
0                                      -315.304382   
1                                        -1.886409   
2                                      -944.744420   
3                                              NaN   
4                                              NaN   
...                                            ...   
2981                                           NaN   

In [14]:
%%bigquery
SELECT DISTINCT demographic_set, demographic_set_category
FROM `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-demographic-rate-cumulative`
ORDER BY demographic_set, demographic_set_category;

Downloading: 100%|██████████| 18/18 [00:01<00:00, 14.61rows/s]


demographic_set                    demographic_set_category
0         Combined                                   Statewide
1          age_gp4                                        None
2          age_gp4                                        0-17
3          age_gp4                                       18-49
4          age_gp4                                       50-64
5          age_gp4                                         65+
6           gender                                           F
7           gender                                           M
8           gender                                           U
9   race_ethnicity                            African American
10  race_ethnicity                             American Indian
11  race_ethnicity                              Asian American
12  race_ethnicity                                      Latino
13  race_ethnicity                                  Multi-Race
14  race_ethnicity  Native Hawaiian and other Pacific Islander
15  race_ethnicity                                       Other
16  race_ethnicity                                     Unknown
17  race_ethnicity                                       White

The covid-19-demographic-rate-cumulative table contains county-levle cross-section data of how cumulative Covid cases/deaths/tests data varies with the demographic variables, which are related to equality.

### 1.4 covid-19-health-equity-metric-pos-30-day-by-cnt 

In [15]:
%%bigquery
SELECT * 
FROM `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-health-equity-metric-pos-30-day-by-cnt`;

Downloading: 100%|██████████| 3540/3540 [00:01<00:00, 2535.27rows/s]


county        date                        metric  metric_value  \
0     Orange  2022-08-13     county_positivity_low_hpi      0.163414   
1     Orange  2022-08-14     county_positivity_low_hpi      0.162770   
2     Orange  2022-08-15     county_positivity_low_hpi      0.161941   
3     Orange  2022-08-16     county_positivity_low_hpi      0.163415   
4     Orange  2022-08-17     county_positivity_low_hpi      0.154994   
...      ...         ...                           ...           ...   
3535  Alpine  2022-09-01  county_positivity_all_nopris      0.333333   
3536  Alpine  2022-09-02  county_positivity_all_nopris      0.500000   
3537  Alpine  2022-09-03  county_positivity_all_nopris      0.500000   
3538  Alpine  2022-09-11  county_positivity_all_nopris      0.500000   
3539   Modoc  2022-09-10  county_positivity_all_nopris      0.000000   

      metric_value_30_days_ago  metric_value_diff  
0                     0.222250          -0.058836  
1                     0.222722          -0.059951  
2                     0.222187          -0.060246  
3                     0.220710          -0.057295  
4                     0.217541          -0.062546  
...                        ...                ...  
3535                  0.000000           0.333333  
3536                  0.000000           0.500000  
3537                  0.000000           0.500000  
3538                  0.000000           0.500000  
3539                  0.000000           0.000000  

[3540 rows x 6 columns]

The covid-19-health-equity-metric-pos-30-day-by-cnt table contains day-by-day county-level data of covid-19 tests positive rate and what's the difference between neighborhoods.
The California Healthy Places Index (HPI) is a composite measure of socioeconomic opportunity applied to census tracts that includes 25 individual indicators across economic, social, education, transportation, housing, environmental and neighborhood sectors. (Race/ethnicity are not variables included in the calculation of the HPI.) Neighborhoods in HPI quartile 1 are labeled as low hpi.

### 1.5 covid-19-race-ethnicity-timeseries 

In [17]:
%%bigquery
SELECT * 
FROM `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-race-ethnicity-timeseries`;

Downloading: 100%|██████████| 2387/2387 [00:01<00:00, 1814.83rows/s]


DATE LOCATION LOCATION_LEVEL           DEMOG  \
0     2022-09-03       CA          state  race_ethnicity   
1     2022-08-27       CA          state  race_ethnicity   
2     2022-08-20       CA          state  race_ethnicity   
3     2022-08-13       CA          state  race_ethnicity   
4     2022-08-06       CA          state  race_ethnicity   
...          ...      ...            ...             ...   
2382  2020-08-01       CA          state  race_ethnicity   
2383  2020-07-25       CA          state  race_ethnicity   
2384  2020-07-18       CA          state  race_ethnicity   
2385  2020-07-11       CA          state  race_ethnicity   
2386  2020-07-04       CA          state  race_ethnicity   

                                       DEMOG_CAT METRIC_CAT            METRIC  \
0                                          White      cases   cases_week_rate   
1                                          White      cases   cases_week_rate   
2                                          White      cases   cases_week_rate   
3                                          White      cases   cases_week_rate   
4                                          White      cases   cases_week_rate   
...                                          ...        ...               ...   
2382  Native Hawaiian and other Pacific Islander     deaths  deaths_week_rate   
2383  Native Hawaiian and other Pacific Islander     deaths  deaths_week_rate   
2384  Native Hawaiian and other Pacific Islander     deaths  deaths_week_rate   
2385  Native Hawaiian and other Pacific Islander     deaths  deaths_week_rate   
2386  Native Hawaiian and other Pacific Islander     deaths  deaths_week_rate   

          VALUE  
0      63.15528  
1      76.77461  
2      87.06567  
3      98.50019  
4     118.29279  
...         ...  
2382    3.68403  
2383    3.68403  
2384    4.42083  
2385    2.21042  
2386    2.21042  

[2387 rows x 8 columns]

In [21]:
%%bigquery
SELECT DISTINCT METRIC
FROM `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-race-ethnicity-timeseries`;

Downloading: 100%|██████████| 3/3 [00:01<00:00,  2.79rows/s]


METRIC
0   cases_week_rate
1   tests_week_rate
2  deaths_week_rate

In [19]:
%%bigquery
SELECT DISTINCT DEMOG, DEMOG_CAT 
FROM `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-race-ethnicity-timeseries`
ORDER BY DEMOG, DEMOG_CAT;

Downloading: 100%|██████████| 7/7 [00:01<00:00,  5.27rows/s]


DEMOG                                   DEMOG_CAT
0  race_ethnicity                            African American
1  race_ethnicity                             American Indian
2  race_ethnicity                              Asian American
3  race_ethnicity                                      Latino
4  race_ethnicity                                  Multi-Race
5  race_ethnicity  Native Hawaiian and other Pacific Islander
6  race_ethnicity                                       White

The ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-race-ethnicity-timeseries table contains weekly rate data of covid-19 cases, tests, and deaths, and how race influences health equity.

## 2. Attributes of Health Inequity

### 2.1 Social determinants of health

In [39]:
%%bigquery
SELECT EXTRACT(MONTH FROM date) AS Month_of_2021, social_det,social_tier,AVG(case_rate_per_100k) AS avg_case_rate_per_100k
FROM `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-case-rate-by-social-det`
WHERE EXTRACT( YEAR FROM date) = 2021
GROUP BY Month_of_2021,social_det,social_tier,sort
ORDER BY Month_of_2021,social_det,sort;

Downloading: 100%|██████████| 288/288 [00:01<00:00, 177.68rows/s]


Month_of_2021 social_det   social_tier  avg_case_rate_per_100k
0                1   crowding  less than 2%               44.695407
1                1   crowding       2% - 5%               56.781871
2                1   crowding      5% - 10%               75.676776
3                1   crowding     10% - 15%               97.046162
4                1   crowding     15% - 20%              115.130432
..             ...        ...           ...                     ...
283             12  insurance      5% - 10%               10.157431
284             12  insurance     10% - 15%               11.496048
285             12  insurance     15% - 25%               12.661453
286             12  insurance     25% - 35%               13.047660
287             12  insurance      over 35%               11.851732

[288 rows x 4 columns]

In [40]:
%%bigquery
SELECT EXTRACT(MONTH FROM date) AS Month_of_2022, social_det,social_tier,AVG(case_rate_per_100k) AS avg_case_rate_per_100k
FROM `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-case-rate-by-social-det`
WHERE EXTRACT( YEAR FROM date) = 2022
GROUP BY Month_of_2022,social_det,social_tier,sort
ORDER BY Month_of_2022,social_det,sort;

Downloading: 100%|██████████| 216/216 [00:01<00:00, 156.39rows/s]


Month_of_2022 social_det   social_tier  avg_case_rate_per_100k
0                1   crowding  less than 2%              130.443291
1                1   crowding       2% - 5%              145.866734
2                1   crowding      5% - 10%              166.359762
3                1   crowding     10% - 15%              190.601379
4                1   crowding     15% - 20%              205.739715
..             ...        ...           ...                     ...
211              9  insurance      5% - 10%               12.221025
212              9  insurance     10% - 15%               13.830448
213              9  insurance     15% - 25%               14.904083
214              9  insurance     25% - 35%               15.364451
215              9  insurance      over 35%               14.886032

[216 rows x 4 columns]

### 2.2 Age and gender

Question: Is there a pattern that shows differences in death rate in 2022 between ages? 

In [29]:
%%bigquery
SELECT demographic_set_category,metric,sum(metric_value) as total_sum
FROM `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-demographic-rate-cumulative`
WHERE demographic_set="age_gp4"
AND county!="California"
GROUP BY demographic_set_category,metric
ORDER BY demographic_set_category,metric

Downloading: 100%|██████████| 15/15 [00:01<00:00,  7.94rows/s]


demographic_set_category  metric  total_sum
0                      None   cases       27.0
1                      None  deaths        0.0
2                      None   tests     1678.0
3                      0-17   cases    22844.0
4                      0-17  deaths        0.0
5                      0-17   tests   432005.0
6                     18-49   cases    72743.0
7                     18-49  deaths        0.0
8                     18-49   tests  1215624.0
9                     50-64   cases    28398.0
10                    50-64  deaths       25.0
11                    50-64   tests   524330.0
12                      65+   cases    23976.0
13                      65+  deaths      310.0
14                      65+   tests   502544.0

According to the data grouping by age, we can see that there is no one age under 49 dead in the past 30 days. However, there are 25 people died whose age are between 50 to 64; and 310 people died whose age are above 65. The death rate for these two groups are 0.088% and 1.293% respectively. We can conclude that the death rate increases with the age.

Question: Is there a pattern that shows differences in death rate in 2022 between gender?

In [30]:
%%bigquery
SELECT demographic_set_category,metric,sum(metric_value) as total_sum
FROM `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-demographic-rate-cumulative`
WHERE demographic_set="gender"
AND county!="California"
GROUP BY demographic_set_category,metric
ORDER BY demographic_set_category,metric

Downloading: 100%|██████████| 9/9 [00:01<00:00,  6.00rows/s]


demographic_set_category  metric  total_sum
0                        F   cases    79644.0
1                        F  deaths      143.0
2                        F   tests  1432300.0
3                        M   cases    67924.0
4                        M  deaths      164.0
5                        M   tests  1192049.0
6                        U   cases     1998.0
7                        U  deaths        0.0
8                        U   tests    73580.0

According to the data, the death rate for female is 0.18%, and the death rate for male is 0.24%. The difference is not that significant enough for us to conclude whether the gender affacts the death rate.

### 2.3  Neighborhoods with different health place index(hpi)

In [27]:
%%bigquery
WITH all_nopris AS(
    SELECT 
        county,
        date,
        metric_value AS positive_rate_all_nopris
    FROM `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-health-equity-metric-pos-30-day-by-cnt`
    WHERE metric = 'county_positivity_all_nopris'
        AND metric_value IS NOT NULL),

 low_hpi AS(
    SELECT 
        county,
        date,
        metric_value AS positive_rate_low_hpi
    FROM `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-health-equity-metric-pos-30-day-by-cnt`
    WHERE metric = 'county_positivity_low_hpi'
         AND metric_value IS NOT NULL)

SELECT 
    all_nopris.county,
    all_nopris.date,
    positive_rate_all_nopris,
    positive_rate_low_hpi,
    (positive_rate_low_hpi - positive_rate_all_nopris) as positive_rate_diff
FROM all_nopris
INNER JOIN low_hpi
ON all_nopris.county = low_hpi.county
    AND all_nopris.date = low_hpi.date

Downloading: 100%|██████████| 1080/1080 [00:01<00:00, 884.55rows/s]


county        date  positive_rate_all_nopris  positive_rate_low_hpi  \
0     Orange  2022-08-13                  0.140707               0.163414   
1     Orange  2022-08-14                  0.139427               0.162770   
2     Orange  2022-08-15                  0.136752               0.161941   
3     Orange  2022-08-16                  0.137453               0.163415   
4     Orange  2022-08-17                  0.132580               0.154994   
...      ...         ...                       ...                    ...   
1075    Napa  2022-09-07                  0.081096               0.093093   
1076    Napa  2022-09-08                  0.083802               0.080000   
1077    Napa  2022-09-09                  0.076966               0.076294   
1078    Napa  2022-09-10                  0.073143               0.084239   
1079    Napa  2022-09-11                  0.077524               0.086721   

      positive_rate_diff  
0               0.022707  
1               0.023343  
2               0.025189  
3               0.025963  
4               0.022414  
...                  ...  
1075            0.011997  
1076           -0.003802  
1077           -0.000671  
1078            0.011096  
1079            0.009197  

[1080 rows x 5 columns]

Neighborhoods with low healthy places index (HPI) tends to have higher positive rate than the county average, indicating more severe Covid epidemic and inadequate testing. Further, the higher the positivity rate, the more significant the differences in positivity rates across regions, while counties with overall lower positivity rates had smaller differences in positivity rates across regions.
The Covid-19 epidemic reveals health inequalities across neighborhoods, and the worse the epidemic, the more significant the inequalities

### 2.4  Race and Ethnicity

In this part,we are going to examine  the case rate and death rate based on different ethnical groups.
This dataset includes COVID-19 related data of California State from 07/04/2020 to 09/10/2022.
The columns of this dataset is explained as follows:
Date: exact date of recording, this dataset takes weekly recorded cases.

Location:The location of the record taken.In our particular dataset, the location is listed as CA.

Location level:The geographic level of the location.In our dataset,the location level is state.

Demographic: The demographic characteristic of all records taken.In our dataset,it is race-ethnity.

Demographic category: The exact division of different ethnicities.We have seven categories: 
white,Latino,Multi-race,Asian-American,American-Indian,African-American and Native Hawaiian.

Metric category:The metric we use when we do the analysis,including number of tests taken, 
number of confirmed cases and number of deaths.

Metric: The metric of the value in our dataset. Cases per week simply indicates number of cases per 100K 
and is not accumulated.

Value:The value based on metrics.



In [1]:
%%bigquery
select*
from `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-race-ethnicity-timeseries`
limit 5

Downloading: 100%|██████████| 5/5 [00:01<00:00,  4.46rows/s]


DATE LOCATION LOCATION_LEVEL           DEMOG DEMOG_CAT METRIC_CAT  \
0  2022-09-03       CA          state  race_ethnicity     White      cases   
1  2022-08-27       CA          state  race_ethnicity     White      cases   
2  2022-08-20       CA          state  race_ethnicity     White      cases   
3  2022-08-13       CA          state  race_ethnicity     White      cases   
4  2022-08-06       CA          state  race_ethnicity     White      cases   

            METRIC      VALUE  
0  cases_week_rate   63.15528  
1  cases_week_rate   76.77461  
2  cases_week_rate   87.06567  
3  cases_week_rate   98.50019  
4  cases_week_rate  118.29279

Above is a brief overview of the data set.

Setup:
We have three measures to compare within each group: tests,cases and death rates. 
Cases and death rates are more related to our topic, so we take these two measures into consideration.
We select a whole year(year 2021) as our target timeline.


Hypothesis:
since the population is not available in the dataset, we would simply assume that the population is consistent within year 2021.Hence, we only need to comoare cases_per_week to analyze the difference within these ethnicities.

In [3]:
%%bigquery
select distinct DEMOG_CAT
from `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-race-ethnicity-timeseries`
#An overview of different ethnicity categories

Downloading: 100%|██████████| 7/7 [00:01<00:00,  5.92rows/s]


DEMOG_CAT
0                                       White
1                                      Latino
2                                  Multi-Race
3                              Asian American
4                             American Indian
5                            African American
6  Native Hawaiian and other Pacific Islander

In [4]:
%%bigquery
select distinct METRIC_CAT
from `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-race-ethnicity-timeseries`
#An overview of different metric categories

Downloading: 100%|██████████| 3/3 [00:01<00:00,  2.32rows/s]


METRIC_CAT
0      cases
1      tests
2     deaths

Question:
Is there a pattern that shows differences in total cases confirmed in 2021 between ethnicities? What may cause this difference ?

In [2]:
%%bigquery
select sum(value) as total_cases_per100K,DEMOG_CAT
from `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-race-ethnicity-timeseries`

where METRIC_CAT = 'cases'
and EXTRACT(year from DATE) = 2021

group by DEMOG_CAT
order by total_cases_per100K desc

Downloading: 100%|██████████| 7/7 [00:01<00:00,  5.92rows/s]


total_cases_per100K                                   DEMOG_CAT
0          12422.54331  Native Hawaiian and other Pacific Islander
1           7091.57208                                      Latino
2           6137.70867                            African American
3           6056.66728                             American Indian
4           4606.08161                                       White
5           4062.35234                                  Multi-Race
6           2989.49561                              Asian American

The above results show significant difference in total cases within different ethnicity groups. The relative high
number of confirmed cases in Native Hawaiian may due to the fact that this ethnicity group lives in a certain area
and is easily get infected if the virus has spread to their living habitat.
The relative low number of confirmed cases in Asian American may due to the fact that they tend to be more cautious facing this virus and did more protective measures compared to other ethnicity groups.

Question:
Is there a pattern that shows differences in total death number in 2021 between ethnicities? 
What may cause this difference ?

In [6]:
%%bigquery
select sum(value) as total_deaths_per100K,DEMOG_CAT
from `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-race-ethnicity-timeseries`

where METRIC_CAT = 'deaths'
and EXTRACT(year from DATE) = 2021

group by DEMOG_CAT
order by total_deaths_per100K desc

Downloading: 100%|██████████| 7/7 [00:01<00:00,  5.45rows/s]


total_deaths_per100K                                   DEMOG_CAT
0             239.46186  Native Hawaiian and other Pacific Islander
1             136.76972                             American Indian
2             134.06235                            African American
3             133.65878                                      Latino
4             109.70330                                       White
5              85.59798                                  Multi-Race
6              81.66826                              Asian American

Overall,the death number is in positive relationship to confirmed cases.The death rate is quite close within
American Indian,African American and Latino,but Latino has more confirmed cases compared to other two ethnities.
It remains unseen and we will discover this phenomena by using graphs and charts.

In [22]:
%%bigquery
select *
from `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-race-ethnicity-timeseries`
where DEMOG_CAT = 'White' and METRIC_CAT = 'cases'
order by value desc
limit 5


Downloading: 100%|██████████| 5/5 [00:01<00:00,  3.43rows/s]


DATE LOCATION LOCATION_LEVEL           DEMOG DEMOG_CAT METRIC_CAT  \
0  2022-01-08       CA          state  race_ethnicity     White      cases   
1  2022-01-15       CA          state  race_ethnicity     White      cases   
2  2022-01-22       CA          state  race_ethnicity     White      cases   
3  2022-01-01       CA          state  race_ethnicity     White      cases   
4  2022-01-29       CA          state  race_ethnicity     White      cases   

            METRIC       VALUE  
0  cases_week_rate  1056.27701  
1  cases_week_rate   899.03200  
2  cases_week_rate   705.59811  
3  cases_week_rate   688.33063  
4  cases_week_rate   425.10535

In [27]:
%%bigquery
select *
from `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-race-ethnicity-timeseries`
where DEMOG_CAT = 'Asian American' and METRIC_CAT = 'cases'
order by value desc
limit 5


Downloading: 100%|██████████| 5/5 [00:07<00:00,  1.56s/rows]


DATE LOCATION LOCATION_LEVEL           DEMOG       DEMOG_CAT  \
0  2022-01-08       CA          state  race_ethnicity  Asian American   
1  2022-01-15       CA          state  race_ethnicity  Asian American   
2  2022-01-01       CA          state  race_ethnicity  Asian American   
3  2022-01-22       CA          state  race_ethnicity  Asian American   
4  2022-01-29       CA          state  race_ethnicity  Asian American   

  METRIC_CAT           METRIC       VALUE  
0      cases  cases_week_rate  1350.28135  
1      cases  cases_week_rate  1003.33249  
2      cases  cases_week_rate   786.22893  
3      cases  cases_week_rate   761.88193  
4      cases  cases_week_rate   453.38421

In [25]:
%%bigquery
select *
from `ba775-team-project-04.ba775_team04_covid_19_equity_metrics_v1kpo4.covid-19-race-ethnicity-timeseries`
where DEMOG_CAT = 'Latino' and METRIC_CAT = 'cases'
order by value desc
limit 5

Downloading: 100%|██████████| 5/5 [00:01<00:00,  4.04rows/s]


DATE LOCATION LOCATION_LEVEL           DEMOG DEMOG_CAT METRIC_CAT  \
0  2022-01-08       CA          state  race_ethnicity    Latino      cases   
1  2022-01-15       CA          state  race_ethnicity    Latino      cases   
2  2022-01-22       CA          state  race_ethnicity    Latino      cases   
3  2021-01-09       CA          state  race_ethnicity    Latino      cases   
4  2022-01-01       CA          state  race_ethnicity    Latino      cases   

            METRIC       VALUE  
0  cases_week_rate  1833.36078  
1  cases_week_rate  1670.06328  
2  cases_week_rate  1266.04039  
3  cases_week_rate   934.17175  
4  cases_week_rate   873.86400

We see that there's a peak in spread of COVID-19 during January 2022.